app/Http/Controllers/Auth/LoginController.php

```diff
@@ -84,6 +84,9 @@ public function showLoginForm()
      */
     public function logout(Request $request)
     {
+        // Invalidates any existing 2FA verified devices.
+        $request->user()->cycleRememberDeviceToken();
```

Comment:
I think I'd like to remove this before going live, although we might want to add a "forget remembered devices" button on the settings page that calls `cycleRememberDeviceToken()`.

Thoughts @tristan-orourke ?